### ISSUE TITLE
Get the following failed to execute goal maven-surefire-plugin Error

### ISSUE THREAD

**Issue by NehalDamania** [kirksl/karate-runner#69]
When using Karate-runner on vs code, with maven project

I am always getting the following error:

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.22.2:test (default-test) on project api_specs: No tests were executed!  (Set -DfailIfNoTests=false to ignore this error.) -> [Help 1]
[ERROR]

However,
when I run the test using VSCode Java Test Runner Plugin, it runs fine.

**Comment by kirksl** [comment]
@NehalDamania i suspect you might have to update your automation project.  can you take a look at our sample/demo project which works with the plugin and confirm that it works on your machine.  if it does i would compare how it differs from your project and update it.  let us know if you are unable to resolve this.

sample project
https://github.com/kirksl/karate-maven-gradle

**Comment by NehalDamania** [comment]
Hi @kirksl , thanks a lot for the quick response. The above sample project 'karate-maven-gradle' is working fine. When running Karate Tests in the sample project in VS Code it shows "OrderRunner" in the VSCode popup as a choice for Karate Runner. However, in my project it does not show the TestRunner that is created exactly same as the OrderRunner from the sample project, instead it shows '_SyncRunner' in VSCode in the popup. 
![image](https://user-images.githubusercontent.com/1287731/96165506-5562f300-0f3a-11eb-9bb0-84ff9ab00a59.png)

This project I had created using the default maven archetype listed in the documentation:

```
mvn archetype:generate \
-DarchetypeGroupId=com.intuit.karate \
-DarchetypeArtifactId=karate-archetype \
-DarchetypeVersion=0.9.6 \
-DgroupId=com.mycompany \
-DartifactId=myproject
```

**Comment by NehalDamania** [comment]
Ok. I got it running.
It was due to _SyncRunner configured by VS Code initially.
I changed the settinsg.json in the .vscode directory of the project to 
```
{
    "karateRunner.karateRunner.default": "TestRunner",
    "files.exclude": {
        "**/.classpath": true,
        "**/.project": true,
        "**/.settings": true,
        "**/.factorypath": true
    },
    "java.configuration.updateBuildConfiguration": "automatic"
}
```
And it ran fine. Thanks @kirksl for the wonderful plugin. I don't know however, how did the vscode set the karateRunner.karateRunner.default to "_SyncRunner" initially. Might be because I didn't have a runner created till then. But, changing this value in the settings.json solved the issue
